app/commands/solution/mentor_discussion/reply_by_mentor.rb

```diff
@@ -3,7 +3,7 @@ class MentorDiscussion
     class ReplyByMentor
       include Mandate
 
-      initialize_with :discussion, :iteration_idx, :content_markdown
+      initialize_with :discussion, :iteration, :content_markdown
```

Comment:
I refactored this as I want to use the iteration in the controller.

Reply:
Hmmm. I changed this to this the other day refactoring it the other way!

I'm ok with this though.